test/html/expect/test_package+ml/Nested/F/index.html

```diff
@@ -27,6 +27,20 @@ <h1>
     <p>
      Some additional comments.
     </p>
+    <h3 id="heading">
```

Comment:
> Some output ordering for functors has changed

I assume that refers to this code, so functor parameters are now documented ahead of the table of contents. Could you comment on why this is so?

Reply:
The output for functor seemed inconsistent with the rest. Semantically, the functor parameters should be part of the "heading" of a page. Practically, there parameters were not in the header but yet outside of the main `<section>` where the rest of the content sits. Apart from functors, the content of a page is always "breadcrumb, heading, toc, content", so it felt natural to do it like that. Given that the toc doesn't reference the functor parameters and is stylistically pushed aside anyway, it shouldn't change anything, so I went ahead.